listener/pkg/v2/event/watcher_event.go

```diff
@@ -40,7 +40,6 @@ func UnmarshalSKREvent(req *http.Request) (*types.WatchEvent, *UnmarshalError) {
 	if err != nil {
 		return nil, &UnmarshalError{"could not read request body", http.StatusInternalServerError}
 	}
-	defer req.Body.Close()
```

Comment:
Hm, why is this not needed anymore?

Reply:
Ok, I seem to have found it. Seems the request body is anyways automatically closed